common-travel.lic

```diff
@@ -81,15 +81,30 @@ module DRCT
     end
   end
 
-  def find_empty_room(search_rooms, idle_room, predicate = nil)
+  def find_empty_room(search_rooms, idle_room, predicate = nil, min_mana = false, strict_mana = false)
+
+    check_mana = min_mana
     loop do
+      found_empty = false
       search_rooms.each do |room_id|
         walk_to(room_id)
         pause 0.1 until room_id == Room.current.id
-        empty = predicate ? predicate.call : (DRRoom.pcs - DRRoom.group_members).empty?
-        return true if empty
+        suitable_room = predicate ? predicate.call : (DRRoom.pcs - DRRoom.group_members).empty?
+        if suitable_room and check_mana and not DRStats.moon_mage?
+          found_empty = true
+          suitable_room = (DRCA.perc_mana >= min_mana)
```

Comment:
This is the comparison that seems odd if `check_mana` (and therefore `min_mana`) is a boolean.